content/post/105-deprecating-secio.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-08-05
+url: /2020-08-05-deprecating-secio/
+title: We're removing support for the SECIO security transport
+author: Jacob Heun
+tags: ipfs, libp2p, secio
+---
+
+## A brief history
```

Comment:
can we start with a very brief "what are security transports and when are they used by IPFS/libp2p" before jumping into the history?

Reply:
I added an overview of security transports and also added a brief intro.

Reply:
perfect